src/lib/one-graph/cli-client.js

```diff
@@ -0,0 +1,278 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable fp/no-loops */
```

Comment:
opened up discussion regarding this lint rule https://github.com/netlify/eslint-config-node/issues/426